clients/lsp-xml.el

```diff
@@ -287,9 +287,9 @@ The value for `enabled' can be always, never or onValidSchema."
   ("xml.catalogs" lsp-xml-catalogs)
   ("xml.trace.server" lsp-xml-trace-server)))
 
-(defconst lsp-xml-jar-version "0.21.0")
+(defconst lsp-xml-jar-version "0.27.0")
 
-(defconst lsp-xml-jar-name (format "org.eclipse.lemminx-%s-uber.jar" lsp-xml-jar-version))
+(defconst lsp-xml-jar-name (format "org.eclipse.lemminx-uber.jar" lsp-xml-jar-version))
```

Comment:
I think we can remove the entire `(format` part.

Reply:
Applied a fix. Thanks for the review and suggestion.